internal/store/driver.go

```diff
@@ -946,8 +946,11 @@ func (d *Driver) calculateHTTPSEdgesFromGateway(edgeMap map[string]ingressv1alph
 								}
 
 								refName := string(backendref.Name)
-								//refNamespace := string(*backendref.Namespace)
-								serviceUID, servicePort, err := d.getEdgeBackendRef(backendref.BackendRef, gtw.Namespace)
+								namespace := httproute.Namespace
+								if backendref.BackendRef.Namespace != nil {
+									namespace = string(*backendref.BackendRef.Namespace)
```

Comment:
> Note that when a namespace different than the local namespace is specified, a ReferenceGrant object is required in the referent namespace to allow that namespace’s owner to accept the reference. See the ReferenceGrant documentation for details.

maybe we shouldn't go there just yet?

Reply:
Good call! I'll check on the `HTTPRoute` namespace!

Reply:
reverted here: https://github.com/ngrok/kubernetes-ingress-controller/pull/356/commits/3bcb295d84e37b89634cde533efa9485fa15d0db